### PR TITLE
PBI-15: Edit Comment

### LINKED ISSUE #42: PBI-15: Creator Can edit comment of announcement in class
**Story Point: 13**
  
  - [x] Text field
  - [x] Button 'Post'
  - [x] Customer can edit the comment they wrote before

### PR DESCRIPTION
**Pull request by IlmaMannix**

### CODE REVIEW

File: src/resources/api/comment.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        }
+
+        data = response.data;
+    } catch (e) {
+        error = e;
+        errorMessage = "Something went wrong, please try again later";
+    }
+
+    return { data, error, errorMessage };
+};
```
Comment (AldiNFitrah) [inline src/resources/api/comment.js]: newline

**Review by AldiNFitrah — APPROVED** [review]
just the minor one, LGTM, thanks!

P.S. I changed the base branch to PBI-14, to prevent the changes from that branch being reviewed here. Please do remember to change it back to staging before merging this PR

**Review by kanziebub — APPROVED** [review]
LGTM, thank you!

**Review by fxdros — APPROVED** [review]
Everything looks good, thanks for your hardwork

**Pull request merged**